tests/test_session_middleware.py

```diff
@@ -3,6 +3,7 @@
 from aiohttp_session import session_middleware
 
 
-async def test_session_middleware_bad_storage():
+async def test_session_middleware_bad_storage() -> None:
     with pytest.raises(RuntimeError):
-        session_middleware(None)
+        # Ignoring typing since return type is on purpose wrong
+        session_middleware(None)  # type: ignore
```

Comment:
Should probably read `Parameter type is wrong on purpose`.
Also, include the code: `# type: ignore[code]`.